src/query/graphite/common/basic_functions.go

```diff
@@ -164,19 +168,21 @@ func Count(ctx *Context, seriesList ts.SeriesList, renamer SeriesListRenamer) (t
 		numSteps := ctx.EndTime.Sub(ctx.StartTime).Minutes()
 		vals := ts.NewZeroValues(ctx, MillisPerMinute, int(numSteps))
 		r := ts.SeriesList{
-			Values: []*ts.Series{ts.NewSeries(ctx, renamer(seriesList), ctx.StartTime, vals)},
+			Values:   []*ts.Series{ts.NewSeries(ctx, renamer(seriesList), ctx.StartTime, vals)},
+			Metadata: block.NewResultMetadata(),
```

Comment:
Here too, seems better to just always force ts.SeriesList to be constructed with some type of constructor.